examples/loop_optimizations_service/opt_loops/opt_loops.cc

```diff
@@ -47,6 +149,13 @@ cl::opt<std::string> InputFilename(cl::Positional, cl::desc("Specify input filen
 cl::opt<std::string> OutputFilename("o", cl::desc("Specify output filename"),
                                     cl::value_desc("filename"), cl::init("-"));
 
+/// Output YAML log file name.
+cl::opt<std::string> OutputYAMLFile("emit-yaml", cl::desc("Specify output YAML log filename"),
+                                    cl::value_desc("filename"), cl::init("/tmp/loops.log"));
+
+// TODO: add other features like "read-yaml", "print-yaml-after-all", "print-yaml-before-all",
```

Comment:
TODOs should be blameable, e.g.

```suggestion
// TODO(mostafaelhoushi): add other features like "read-yaml", "print-yaml-after-all", "print-yaml-before-all",
```

or:

```suggestion
// TODO(github.com/facebookresearch/CompilerGym/issues/553): add other features like "read-yaml", "print-yaml-after-all", "print-yaml-before-all",
```

This applies here and elsewhere in this PR.